templates/zerver/desktop_redirect.html

```diff
@@ -3,12 +3,14 @@
 
 {% block content %}
 <div class="flex">
-    <div class="desktop-redirect-box">
+    <div class="desktop-redirect-box white-box">
         <img class="avatar desktop-redirect-image" src="{{ realm_icon_url }}" alt=""/><br>
-        <p>{% trans %}Copy this login token and return to your Zulip app to finish logging in:{% endtrans %}</p>
-        <p>
-            <input id="desktop-data" value="{{ desktop_data }}" readonly />
-            <button id="copy" tabindex="0" class="btn btn-primary" data-clipboard-target="#desktop-data">{% trans %}Copy{% endtrans %}</button>
+        <p class="copy-token-info">{% trans %}Copy this login token and return to your Zulip app to finish logging in:{% endtrans %}</p>
+        <p class="new-style">
```

Comment:
does `new-style` belong here, or higher up on the `flex` class div or white-box?  Generally we have it as an outer class because at some point we'll just remove it and make it the default styling.